<ui version="4.0" >
 <author>Niko Sams</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VocableCoach</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Search:</string>
        </property>
        <property name="buddy" >
         <cstring>filterLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLineEdit" />
      </item>
      <item>
       <widget class="QComboBox" name="boxSelectionCombo" >
        <property name="toolTip" >
         <string>Box-Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="vocableEditorView" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionProperties" />
    <addaction name="actionQuiz" />
    <addaction name="separator" />
    <addaction name="actionPrintPreview" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionImport" />
    <addaction name="actionExport" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionSelectAll" />
    <addaction name="actionModifyLesson" />
    <addaction name="actionResetBox" />
    <addaction name="separator" />
    <addaction name="actionAddVocable" />
    <addaction name="actionEditVocable" />
    <addaction name="actionDeleteVocable" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionAddVocable" />
   <addaction name="actionEditVocable" />
   <addaction name="actionDeleteVocable" />
   <addaction name="separator" />
   <addaction name="actionQuiz" />
  </widget>
  <action name="actionNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImport" >
   <property name="text" >
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport" >
   <property name="text" >
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="actionAddVocable" >
   <property name="text" >
    <string>&amp;Add Vocable</string>
   </property>
   <property name="shortcut" >
    <string>Ins</string>
   </property>
  </action>
  <action name="actionEditVocable" >
   <property name="text" >
    <string>&amp;Edit Vocable</string>
   </property>
  </action>
  <action name="actionDeleteVocable" >
   <property name="text" >
    <string>&amp;Delete Vocable</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="actionQuiz" >
   <property name="text" >
    <string>&amp;Quiz</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About VocableCoach</string>
   </property>
  </action>
  <action name="actionProperties" >
   <property name="text" >
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="actionPrintPreview" >
   <property name="text" >
    <string>Print Preview...</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionModifyLesson" >
   <property name="text" >
    <string>&amp;Modify Lesson</string>
   </property>
  </action>
  <action name="actionResetBox" >
   <property name="text" >
    <string>Reset &amp;Box</string>
   </property>
  </action>
  <action name="actionSelectAll" >
   <property name="text" >
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut" >
    <string>Strg+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
